.github/workflows/test.yml

```diff
@@ -17,6 +17,5 @@ jobs:
           node-version: 'lts/*'
           cache: 'pnpm'
       - run: pnpm install
-      - run: pnpm run test:format
-      - run: pnpm run test:types
-      - run: pnpm run test:lint
+      - run: pnpm run test
```

Comment:
Hm, how about running test:* sequentially? I think it's easier to know which one stops.